packages/lesswrong/components/dialogues/DialogueRecommendationRow.tsx

```diff
@@ -218,6 +231,39 @@ const ExpandCollapseText = ({classes, isExpanded, numHidden, toggleExpansion}: E
   )
 };
 
+interface CommentViewProps {
+  comment: {
+    _id: string;
+    postId: string;
+    contents: {
+      html: string;
+      plaintextMainText: string;
+    };
+  };
+  classes: ClassesType<typeof styles>;
+}
+
+const CommentView: React.FC<CommentViewProps> = ({ comment, classes }) => {
+
+  const { PostsTooltip } = Components
+
+  const { document: post, loading, error } = useSingle({
+    collectionName: "Posts",
+    fragmentName: 'PostsPage',
+    documentId: comment.postId,
+    allowNull: true,
+  });
+
+  if (loading) return <div>Loading...</div>;
```

Comment:
I think we should strongly prefer to use the `Loading` component, which is already used everywhere else and has a less janky transition from a loading state to a non-loading state